.chloggen/mdatagen-usage.yaml

```
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

```suggestion
change_type: enhancement
```

Doesn't seem a bug to me